wezterm-ssh/Cargo.toml

```diff
@@ -20,15 +23,21 @@ log = "0.4"
 portable-pty = { version="0.5", path = "../pty" }
 regex = "1"
 smol = "1.2"
-ssh2 = {version="0.9", features=["openssl-on-win32"]}
-#ssh2 = {version="0.9"}
+# ssh2 = {version="0.9", features=["openssl-on-win32"]}
```

Comment:
I can't merge this PR without `openssl-on-win32` without breaking wezterm on windows.
Would it make things saner for you while iterating with `distant` if this dep directly referenced my fork instead of using the patched version in the top level `Cargo.toml`?

Reply:
I wasn't planning to leave this bit once everything passes :) Was just so I could test quickly before you mentioned the patched version you've got! I'll switch over to the patched version myself.

Reply:
Reverted back to including `openssl-on-win32`.